Component/Console/Style/ImagineStyle.php

```diff
@@ -146,8 +147,11 @@ private function compileString(string $format, array $replacements = []): string
             return $format;
         }
 
-        if (false !== $compiled = @vsprintf($format, $replacements)) {
-            return $compiled;
+        try {
+            if (false !== $compiled = @vsprintf($format, $replacements)) {
+                return $compiled;
+            }
+        } catch (ValueError $error) {
```

Comment:
I decided to catch the error instead of changing the tests, to keep the contract consistent across PHP versions.